Recycle/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Recycle</title>
+    <link rel="icon" href="./img/glow.png" />
+  </head>
+  <body>
+    <div class="container start-animation">

```

Comment:
`start-animation` class has to be added after `DOMContentLoaded` event is performed